specificatie/openapi.yaml

```diff
@@ -0,0 +1,377 @@
+        '401':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/401"
+        '403':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/403"
+        '404':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/404"
+        '406':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/406"
+        '500':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/500"
+        '501':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/501"
+        '503':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/503"
+        'default':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/default"
+      tags:
+        - Historie
+components:
+  parameters:
+    burgerservicenummer:
+      $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/v1.0.0/specificatie/openapi.yaml#/components/parameters/burgerservicenummer"
+  schemas:
+    NationaliteithistorieHalCollectie:
+      type: object
+      properties:
+        _links:
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/HalCollectionLinks"
+        _embedded:
+          $ref: '#/components/schemas/NationaliteithistorieHalCollectieEmbedded'
+    NationaliteithistorieHalCollectieEmbedded:
+      type: object
+      properties:
+        nationaliteithistorie:
+          type: array
+          items:
+            $ref: '#/components/schemas/Nationaliteithistorie'
+    Nationaliteithistorie:
+      allOf:
+      - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/v1.0.0/specificatie/openapi.yaml#/components/schemas/Nationaliteit"
+      - description: |
+                      Gegevens over de verblijfsrechtelijke status van de ingeschrevene.
+                      * **datumEinde**: Datum waarop de geldigheid van de gegevens over de Nationaliteit is beëindigd.
+                      * **datumIngang**: Datum waarop de gegevens over de Nationaliteit geldig zijn geworden.
+                      * **aanduiding** : Nationaliteittabel, die aangeeft over welke verblijfsrechtelijke status de ingeschrevene beschikt.
```

Comment:
Ook deze beschrijving klopt niet. Het gaat hier om de code en omschrijving van de nationaliteit die de persoon heeft

Reply:
deze 3 velden komen niet voor in de nationaliteit. 

Wel zijn er:
- datumEindeGeldigheid (i.p.v. datumEinde)
- datumIngangGeldigheid (i.p.v. datumIngang)
- nationaliteit (i.p.v. aanduiding)
- redenOpname
- redenBeeindigen

Gelukkig zijn deze propertynamen behoorlijk zelfverklarend, dus betwijfel dat een aanvullende description veel toe gaat voegen

Reply:
Aangepast. Tot nu toe hebben we voor alle elementen die een datumOnvolledig hebben en alle elementen die op een waardetabel gebaseerd zijn een toelichting gedaan. Dat heb ik nu ook aangehouden, maar als dat anders moet hoor ik dat wel.